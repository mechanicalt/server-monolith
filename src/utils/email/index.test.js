// @flow

import { Email } from './';

describe('emailUtils', () => {
  it('Email', () => {
    const email = new Email('createOffer', {to: 'example@menternship.org', subject: 'hello'}, {});
    return email.prepare().then((finalEmail) => {
      expect(finalEmail.envelope).toMatchSnapshot();
    });
  });
});
