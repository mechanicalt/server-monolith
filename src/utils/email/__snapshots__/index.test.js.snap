// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`emailUtils acceptOffer 1`] = `
Object {
  "from": "noreply@menternship.org",
  "to": Array [
    "example@menternship.org",
  ],
}
`;

exports[`emailUtils createOffer 1`] = `
Object {
  "from": "noreply@menternship.org",
  "to": Array [
    "example@menternship.org",
  ],
}
`;

exports[`emailUtils rejectOffer 1`] = `
Object {
  "from": "noreply@menternship.org",
  "to": Array [
    "example@menternship.org",
  ],
}
`;
